src/main/java/com/scalar/database/storage/cassandra4driver/ClusterManager.java

```diff
@@ -0,0 +1,106 @@
+package com.scalar.database.storage.cassandra4driver;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.api.core.metadata.schema.TableMetadata;
+import com.google.common.annotations.VisibleForTesting;
+import com.scalar.database.config.DatabaseConfig;
+import com.scalar.database.exception.storage.ConnectionException;
+import com.scalar.database.exception.storage.StorageRuntimeException;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
+import javax.annotation.concurrent.ThreadSafe;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A cluster manager
+ *
+ * @author Hiroyuki Yamada
```

Comment:
It should be 
```suggestion
 * @author Yuji Ito, Hiroyuki Yamada
```
Or we can delete this line and delegate it to commit logs.
(Sorry all the initial codes include this author annotation since it is automatically added by the editor I use...)

Reply:
Thanks!